File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] a, int[] b) {
        int k=0;
        int c[]=new int[a.length+b.length] ;
        for( int i =0 ;i<a.length+b.length;i++){
            if(i<a.length){
            c[i]=a[i];}
            else if (i>=a.length){
                c[i]=b[k];
                k+=1;
            }
        }

        Arrays.sort(c);

return c.length%2==0 ? ( (double)(c[c.length/2]+c[(c.length-1)/2])/2 ): (double)c[c.length/2];    
    }
}

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& a, vector<int>& b) {
        vector<int> v;
        int i=0,j=0;
        int n = a.size() , m = b.size();
        while(i< n && j <m){
            if(a[i]<b[j]){
                v.push_back(a[i++]);
            }
            else {
                v.push_back(b[j++]);
            }
        }
        while(i<n){
            v.push_back(a[i++]);
        }
        while(j<m){
            v.push_back(b[j++]);
        }
        return (n+m)%2==0 ? (double)(v[(n+m)/2] + v[(n+m)/2 -1])/2 : v[(n+m)/2];
    }
};